### ISSUE TITLE
Pretrained model on objaverse

### ISSUE THREAD

**Issue by thuliu-yt16** [salesforce/ULIP#40]
Will it be released? If true, when?

**Comment by Tycho-Xue** [comment]
@thuliu-yt16, sorry for the late reply, I was out of the office until this week.
for the ULIP-2 objaverse pre-trained model, it has been released in this already, check out the news part.

**Comment by auniquesun** [comment]
@Tycho-Xue According to my understanding, the released pretrained model of ULIP-2 is pointbert.pt, but the version of pointnext.pt is absent. Could you please also release it?

**Comment by thuliu-yt16** [comment]
Indeed! I didn't see whether these released models are trained on shapenet or objaverse or just both. @Tycho-Xue Could you please clarify it?

**Comment by Tycho-Xue** [comment]
@auniquesun , @thuliu-yt16 , sorry for the delay, I just uploaded it here in the provided gcp bucket. It is the base version of pointnext (1.4M params), pre-trained on Objaverse using the generated holistic captions.
"sfr-ulip-code-release-research/pretrained_models/pointnext_pretrained"
https://storage.cloud.google.com/sfr-ulip-code-release-research/pretrained_models/pointnext_pretrained/ULIP_2-pointnext_objaverse.pt
